toolchains/local_java_repository.bzl

```diff
@@ -202,7 +197,7 @@ local_java_runtime(
     java_home = "%s",
     version = "%s",
 )
-""" % (repository_ctx.name, runtime_name, java_home, version)
```

Comment:
Just one question out of curiosity. repository_ctx.attr.name is the same as repository_ctx.name?

Reply:
yep